db/seeds.rb

```diff
@@ -1,7 +1,124 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+RANDOM_BOOLEAN = [true, false]
+
+AMOUNT_ISSUES = 50
+AMOUNT_LINES = 10
+AMOUNT_SETTINGS = 50
+AMOUNT_STOPS = 50
+AMOUNT_STOPS_USERS = 50
+AMOUNT_USERS = 50
+AMOUNT_VEHICLES = 50
+
+LIST_ISSUES_TYPES = [
+                      "Delays",
+                      "Cleanliness",
+                      "Safety",
+                      "Crowding",
+                      "Accessibility",
+                      "Staff",
+                      "Other"
+                    ]
+
+LIST_SYSTEM_TYPES = [1,2,3,4]
+
+LIST_LINE_COLORS =  [
```

Comment:
Colors should be hex codes, not names, since they are used for CSS display. I'll fix this